instance/github.py

```diff
@@ -148,6 +149,12 @@ class PR:
     """
     Representation of a GitHub Pull Request
     """
+    SETTINGS = {
+        'DATABASE_URL': 'DATABASE_URL',
+        'MONGO_URL': 'MONGO_URL',
+        'EPHEMERAL_DATABASES': 'EPHEMERAL_DATABASES',
+    }
```

Comment:
I think you don't need this map.  The setting names only appear a single time in the properties below.  The properties already form an abstraction layer, so we don't need another level of abstraction.  (If you want to keep the dict anyway for some reason, make sure to update the tests to use this dict instead of hard-coding the setting names there.)